fenix/app/src/main/java/org/mozilla/fenix/components/toolbar/BrowserToolbarView.kt

```diff
@@ -80,13 +81,23 @@ class BrowserToolbarView(
 
         with(context) {
             val isPinningSupported = components.useCases.webAppUseCases.isPinningSupported()
+            val searchUrlBackground = if (IncompleteRedesignToolbarFeature(context.settings()).isEnabled) {
+                R.drawable.search_new_url_background
```

Comment:
Instead of calling this `search_new_url_background`, it might be better to just rename the existing one `old_search_url_background` so that we can just delete it later and not have to rename it.

Reply:
Smart. Thank you